scalalib/overrides-2.12/scala/collection/immutable/VM.scala

```diff
@@ -5,6 +5,7 @@ import scala.scalanative.libc.stdatomic.memory_order._
 
 // Backport from scala.runtime moved into s.c.immutable and made package
 // private to avoid the need for MiMa allowlisting.
-/* private[immutable] */ object VM {
+/* private[immutable] */
```

Comment:
Please add an exclusion rule for the whole`scalalib` directory - nothing in here should be formatted to reduce the diff when comparing our overrides with the original Scala standard library sources 

We'd also need to revert all changes made in this directory